### ISSUE TITLE
Need to work with Python 3.8+

### ISSUE THREAD

**Issue by Psijic** [TylerGubala/blenderpy#130]
It's currently hard to install Python 3.7 in OpenSuse TW, so need to update the **bpy**.

**Comment by abhimanyuchadha96** [comment]
@Psijic Were you able to build it on Python 3.8 or higher?

**Comment by Psijic** [comment]
> @Psijic Were you able to build it on Python 3.8 or higher?

No, even after some hacks were implemented. Also they released a new version with modern Python support later.
